openapi2conv/openapi2_conv.go

```diff
@@ -107,8 +111,7 @@ func ToV3(doc2 *openapi2.T) (*openapi3.T, error) {
 
 	doc3.Security = ToV3SecurityRequirements(doc2.Security)
 	{
-		sl := openapi3.NewLoader()
-		if err := sl.ResolveRefsIn(doc3, nil); err != nil {
+		if err := loader.ResolveRefsIn(doc3, location); err != nil {
 			return nil, err
 		}
 	}
```

Comment:
Can you drop the surrounding braces as well please?

Reply:
Done